src/spootnik/reporter/impl.clj

```diff
@@ -372,16 +371,15 @@
 (defn parse-pggrouping-keys [grouping-keys]
   (into {} (for [[k v] grouping-keys] [(csk/->snake_case_string k) v])))
 
-;; "sentry" is a sentry map like {:dsn "..."}
-;; "raven-options" is the options map sent to raven http client
-;; http://aleph.io/codox/aleph/aleph.http.html#var-request
+;; "sentry" is a configuration map sent to sentry.io on initialisation
```

Comment:
When migrating to the new reporter, we will have to provide the environment and release, instead of reading it off from `SENTRY_*` env vars right?

Reply:
don't get this

Reply:
I mean that we may want to update the README to say clients should specifically pass environment and release when sending events.
Eg blockproxy was relying on `SENTRY_` env vars: https://github.com/exoscale/blockstorage/blob/master/.kargo/collector.edn#L36 and `io.sentry.logback.SentryAppender`, which would read off those env vars; when we move to the new reporter, we have to explicitly pass these variables to the sentry config